src/web/browser/sentry/detectAdBlocker.ts

```diff
@@ -38,7 +38,7 @@ export const isAdBlockInUse: () => Promise<boolean> = () => {
                     displayProp === 'none' ||
                     !!(
                         mozBindingProp &&
-                        mozBindingProp.indexOf('about:') !== -1
+                        mozBindingProp.includes('about:')
```

Comment:
🎉

Reply:
Yeah this and startsWith were really nice changes!